packages/components/src/components/tab/tab.stories.ts

```diff
@@ -68,9 +95,36 @@ export const Variant = {
 
 export const Active = {
   render: () => html`
-    <div class="flex gap-12">
-      <sd-tab active>Default</sd-tab>
-      <sd-tab variant="container" active>Container</sd-tab>
+    <div class="grid grid-cols-2 gap-12">
```

Comment:
can we have have the second tab activated? Now it actually looks exactly the same as the story above and therewith its not directly clear to the users, what "active" is actually doing. 
<img width="1254" alt="image" src="https://github.com/user-attachments/assets/87ffb381-2df5-418c-8138-e8692aec8669">

Reply:
@karlbaumhauer I totally agree with you and that was my first approach but for some reason it just ignores the "active" on second tab and makes the first active

Reply:
thats strange. Pls have a look into this bug.

Reply:
Fixed. There is still an issue but a bugfix ticket for that is created: https://github.com/solid-design-system/solid/issues/1455.